### ISSUE TITLE
Straight line correction 🪄 

### ISSUE THREAD

**Issue by MusicFreak456** [WizzWriters/spelldraw-app#137]
Although it poses some significant difficulties, I think it would be the most useful figure to correct.

**Comment by MusicFreak456** [comment]
@ventus550  Any thoughts about feasibility?

**Comment by ventus550** [comment]
Actually, I believe this to be the easiest problem yet :)
Gather the points and fit a line, in say linear regression fashion, then check the error.
Should it happen to be large we abort the correction, otherwise we accept the model's hypothesis.

**Comment by ventus550** [comment]
I was also wondering if we could make this work for polynomials too. The tricky part is that we do not have the data and thus we can't simply do classification. One way would be to integrate it into the ShapeWizard where we would first test for the `other` cathegory and should the image pass such test we could then attempt some kind of polynomial fit with some bias towards linearity.

**Comment by MusicFreak456** [comment]
Yeah, seems straight forward. I don't believe it can be the actual linear regression(?). Or maybe it can if we treat time as one of the dimensions.
